AdvanceSpecFlowProject/SpecFlowProject/StepDefinitions/LanguageFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,379 @@
+using NUnit.Framework;
+using RazorEngine;
+using SpecFlowProject.Hooks;
+using SpecFlowProject.Model;
+using SpecFlowProject.Steps;
+using SpecFlowProject.Steps.LanguageSteps;
+using SpecFlowProject.Utilities;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowProject.StepDefinitions
+{
+    [Binding]
+    public class LanguageFeatureStepDefinitions : CommonDriver
+    {
+        ActionOnLanguageComponentSteps ActionOnLanguageStepsObj;
+        ProfileLanguageOverviewComponentSteps ProfileLanguageOverviewStepsObj;
+
+        public LanguageFeatureStepDefinitions()
+        {
+            ActionOnLanguageStepsObj = new ActionOnLanguageComponentSteps();
+            ProfileLanguageOverviewStepsObj = new ProfileLanguageOverviewComponentSteps();
+        }
+
+        [When(@"I added a language '([^']*)'")]
+        public void WhenIAddedALanguage(string userLanguage)
+        {
+            //-- Add language--
+            try
+            {
+                List<LanguageModel> languageList = JsonReader.GetData<LanguageModel>(userLanguage);
+
+                //Deleting language
```

Comment:
I would move the delete of all languages and add in to a method

Reply:
Issue resolved. Please review.